### ISSUE TITLE
fix: 🤔 sd-select placeholder font color is too light

### ISSUE THREAD

**Issue by MartaPintoTeixeira** [solid-design-system/solid#1539]
## Current behavior
The sd-select Placeholder font-color is diferent in code and in figma.
![Image](https://github.com/user-attachments/assets/2c02c59e-d53c-4f05-b554-172f7fe370b4)

## Expected behavior
In figma sd-select Placeholder font-color is neutral-700
![Image](https://github.com/user-attachments/assets/c4ab41e8-8a25-474b-bc8f-bb91f903a902)

## Steps to reproduce
Open forms template and compare select item in figma and storybook.

## DoR
- [x] Item has business value
- [x] Item has been estimated by the team
- [x] Item is clear and well-defined
- [x] Item dependencies have been identified

## DoD
- [ ] Documentation has been created/updated (if applicable)
- [ ] Migration Guide has been created/updated (if applicable)
- [ ] Relevant E2E tests (Features, A11y, Bug fixes) are created/updated
- [ ] Relevant stories (Features, A11y) are created/updated
- [ ] Implementation works successfully on `feature` branch

**Comment by coraliefeil** [comment]
@mpantuso you did not mention this color issue as far as I can see. Or?

Should be fixed within the a11y issues though. So I labeled it with "A11y Audit" @yoezlem

**Comment by auroraVasconcelos** [comment]
Closed by fix/change-sd-select-placeholder-text-color-to-neutral-700
